odilia/src/tower.rs

```diff
@@ -405,15 +423,16 @@ where
 // S: is some state type that implements Clone
 // Fut: is a future whoes output is Result<Response, Error>, and which is sendable across threads
 // statically
-impl<F, Fut, S, T1, T2, E> Handler<(Request, T1, T2), S, E> for F
+impl<F, Fut, S, T1, T2, E, R> Handler<(Request, T1, T2), S, E> for F
 where
 	F: FnOnce(E, T1, T2) -> Fut + Clone + Send,
-	Fut: Future<Output = Result<Response, Error>> + Send + 'static,
+	Fut: Future<Output = Result<R, Error>> + Send + 'static,
 	S: Clone,
 	T1: From<S>,
 	T2: From<S>,
+	R: IntoCommands,
```

Comment:
aha, so we do the suggested thing here, instead of there? then, what's that module for?

Reply:
Fixed as discussed above.